pyproject.toml

```diff
@@ -5,5 +5,9 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[tool.ruff.lint]
```

Comment:
I think it makes sense to include `isort` and `pylint` as well (plus the minor adjustments shown here):
```
[tool.ruff.lint]
# Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
# McCabe complexity (`C901`) by default.
# Adds isort ("I") and pylint ("PL") to defaults.
select = ["E4", "E7", "E9", "F", "I", "PL"]
# Ignore pylint's overwritten loop variable by assignment error
ignore = ["PLW2901"]

[tool.ruff.lint.pylint]
max-args = 10
```

Reply:
@runame please check if you're OK with this:

1. Only enable "PLE" and "PLW" ("PLR" is disabled)
2. Quote-style: "double" --- just follow `black` as is